playbook.yml

```diff
@@ -48,3 +48,5 @@
       when: "'setup_pemagent' in lookup('edb_devops.edb_postgres.supported_roles', wantlist=True)"
     - role: autotuning
       when: "'autotuning' in lookup('edb_devops.edb_postgres.supported_roles', wantlist=True)"
+    - role: manage_patches
+      when: "'setup_efm' in lookup('edb_devops.edb_postgres.supported_roles', wantlist=True)"
```

Comment:
`when: "'manage_patches' in lookup('edb_devops.edb_postgres.supported_roles', wantlist=True)"`
and please update the `supported_roles` lookup function

Reply:
Fixed